libpam/pam_misc.c

```diff
@@ -158,14 +159,14 @@ char *_pam_memdup(const char *x, int len)
 
 int _pam_mkargv(const char *s, char ***argv, int *argc)
 {
-    int l;
+    size_t l;
     int argvlen = 0;
     char *sbuf, *sbuf_start;
     char **our_argv = NULL;
     char **argvbuf;
     char *argvbufp;
 #ifdef PAM_DEBUG
-    int count=0;
+    unsigned count=0;
```

Comment:
Why not use `size_t` here, as this is a counter for the number of some items?

Reply:
The `argc` is an `int` and cannot be changed, so the `unsigned` int counterpart will definitely be large enough. Also we have the `%u` formatter which is universally available for it.